envoy/rockcraft.yaml

```diff
@@ -0,0 +1,52 @@
+# Based on: https://github.com/envoyproxy/envoy/blob/release/v1.27/ci/Dockerfile-envoy
+name: envoy
+summary: Envoy is an open-source edge and service proxy designed for cloud-native applications.
+description: |
+  Envoy is essential for modern applications, particularly those built using microservices, 
+  by providing reliable, efficient, and secure service communication along with valuable observability features.
+version: "1.27.0"
```

Comment:
We should use 1.27.3 as upstream [kubeflow pipelines 2.2.0 do](https://github.com/kubeflow/pipelines/blob/2.2.0/third_party/metadata_envoy/Dockerfile#L15)